pydantic/utils.py

```diff
@@ -118,6 +121,50 @@ def almost_equal_floats(value_1: float, value_2: float, *, delta: float = 1e-8)
     return abs(value_1 - value_2) <= delta
 
 
+def generate_model_signature(
+    init: Callable[..., None], fields: Dict[str, 'ModelField'], config: Type['BaseConfig']
+) -> Signature:
+    """
+    Generate signature for model based on its fields
+    """
+
+    present_params = signature(init).parameters.values()
+    merged_params: OrderedDict[str, Parameter] = OrderedDict()
+    var_kw = None
+    use_var_kw = False
+
+    for param in islice(present_params, 1, len(present_params)):  # skip self arg
+        if param.kind is param.VAR_KEYWORD:
+            var_kw = param
+            continue
+        merged_params[param.name] = param
+
+    allow_names = config.allow_population_by_field_name
+    for field_name, field in fields.items():
+        param_name = field.alias
```

Comment:
no need for this assignment I don't think.

Reply:
Why? It can be changed later to `field_name` if it's not correct identifier and field names are allowed, and used more than once.

Reply:
Okay.